nextflow.config

```diff
@@ -334,7 +334,7 @@ manifest {
 
 // Nextflow plugins
 plugins {
-    id 'nf-schema@2.3.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
+    id 'nf-schema@2.2.0' // Validation of pipeline parameters and creation of an input channel from a sample sheet
```

Comment:
the failing CI was fixed and this downgrade is not necessary anymore, you can add it back if you want :)